resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml

```diff
@@ -11,3 +11,7 @@ rules:
   - apiGroups: ["authentication.k8s.io"]
     resources: ["tokenreviews"]
     verbs: ["create"]
+  # XXX: remove before 2.1 release
```

Comment:
Link the Jira issue here, and link this line in the jira issue :)

Reply:
done

Reply:
wasn't this solved by https://github.com/maistra/istio/pull/283?

Reply:
Unfortunately, no. :(

Reply:
ahh got it, only now read the issue comments. thanks!